### DISCUSSION TITLE
Remapping shift + shift

### DISCUSSION THREAD

**Discussion by wiiznokes** [helix-editor/helix#10285] [Q&A]
Is it possible to remap two shift ?
And also, how can i remap ENTER ?

**Comment by the-mikedavis** [comment]
Remapping left and right shift together is not supported. Currently `S-x` means either left or right shift and x. For the Enter key you can use `ret`: https://docs.helix-editor.com/master/remapping.html#special-keys-and-modifiers

**Reply by wiiznokes** [reply]
I meant two left shift in a row. Like IDEA.

**Reply by the-mikedavis** [reply]
That wouldn't be possible either. We treat shift only as a modifier - we don't handle key events for presses / releases
